plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/crysl/templates/stringhashing/StringHasher.java

```diff
@@ -16,20 +16,22 @@
 
 import de.cognicrypt.codegenerator.crysl.CrySLCodeGenerator;
 
+

```

Comment:
I suggest adding a simple Javadoc for the class, as you have done for the other templates as well. Also, which task in CogniCrypt GEN, generates this template? I could not find it myself.

Reply:
Unfortunately, I could not find a use case in the codegen wizard that leads to the respective template. I talked to Krishna about it, and we decided to leave it for now. That's why I did not add any java docs here.

Reply:
Oh OK. I also could not find a use case in the wizard where this wizard is generated. Maybe we can decide to remove it in the future.